site/content/docs/snippets/_index.md

```diff
@@ -68,27 +87,27 @@ Browser checks to have granular feedback whether each part functions.
 Create a snippet named `github_login` in the "code snippets" section with a function that executes the login routine.
 
 {{< tabs "github_login" >}}
-{{< tab "Playwright" >}}
-```javascript
+{{< tab "TypeScript" >}}
+```ts
 async function gitHubLogin (page, username, password) {
-  await page.goto("https://github.com/login")
-  await page.type("#login_field", username)
-  await page.type("#password", password)
-  await page.click('[name="commit"]')
+  await page.goto('https://github.com/login')
+  await page.locator('#login_field').type(username)
```

Comment:
Would be nice to use the recommended locators `getByX()` where possible. 

We could use `page.getByLabel('Password')`, `page.getByLabel('Username or email address')`, and `page.getByRole('button', { name: 'Sign in' })`

Reply:
I agree but none of these will work if we want to stay true to the original page. The inputs have no label or placeholder and the "sign in" button is no button but another input element. So either we use `locator` or we pick a different example website.

<img width="1202" alt="Screenshot 2023-02-16 at 13 27 48" src="https://user-images.githubusercontent.com/54396648/219377567-a76d674e-822e-485b-bdeb-9e88615e82f5.png">